components/SvgCard.vue

```diff
@@ -2,7 +2,11 @@
   <data-view class="SvgCard" :title="title" :title-id="titleId" :date="date">
     <template v-slot:button>
       <p class="Graph-Desc">
-        {{ $t('（注）都内において疑い例または患者の濃厚接触者として検査を行ったものについて掲載') }}<br />
+        {{
+          $t(
+            '（注）都内において疑い例または患者の濃厚接触者として検査を行ったものについて掲載'
+          )
+        }}<br />
         （{{ $t('チャーター機帰国者、クルーズ船乗客等は含まれていない。') }}）
```

Comment:
ここの括弧なんですが、おそらく欧米語だと半角になるべきなので、翻訳辞書の方に括弧まで含めてもらってもよいでしょうか？
```suggestion
        {{ $t('（チャーター機帰国者、クルーズ船乗客等は含まれていない。）') }}
```

Reply:
@MaySoMusician あー確かにそうですね．

Reply:
@MaySoMusician 変更しました